### DISCUSSION TITLE
Live cycle schema

### DISCUSSION THREAD

**Discussion by existe-deja** [nuxt-community/auth-module#1602] [Ideas & Feature Requests]
### What problem does this feature solve?

Having more documentation halps a lot to better understand how things works. In my particular case, I want to know when the user is fetched on the reloading of a page so I can fetch additional data. Before I did it in nuxtServerInit store action.

### What does the proposed changes look like?

A typical lifecycle schema.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c692">#c692</a>)</em></sub></div>

**Comment by bmulholland** [comment]
Thanks for the idea. Can you share an example of what you're thinking of?

**Comment by existe-deja** [comment]
Hi,

I was thinking of this kind of things > https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram
One diagram per scheme could be very helpful.
